config/routes.rb

```diff
@@ -1,10 +1,8 @@
-# frozen_string_literal: true
-
 Rails.application.routes.draw do
-  root 'recipes#public_recipes'
+  root 'recipes#index'
```

Comment:
This will give error because according to the project description;
Recipies list: `Should display a list of recipes created by the logged-in user as in the wireframe.`
If the user is not logged in, it will render empty page.
I suggest to set `public recipes` as root page as it was before.